src/riak_core_util.erl

```diff
@@ -704,6 +712,124 @@ proxy(Parent, Fun) ->
             ok
     end.
 
+-spec enable_job_class(atom(), atom()) -> ok | {error, term()}.
+%% @doc Enables the specified Application/Operation job class.
+%% This is the public API for use via RPC.
+enable_job_class(Application, Operation)
+        when erlang:is_atom(Application) andalso erlang:is_atom(Operation) ->
+    enable_job_class({Application, Operation});
+enable_job_class(Application, Operation) ->
+    {error, {badarg, {Application, Operation}}}.
+
+-spec disable_job_class(atom(), atom()) -> ok | {error, term()}.
+%% @doc Disables the specified Application/Operation job class.
+%% This is the public API for use via RPC.
+disable_job_class(Application, Operation)
+        when erlang:is_atom(Application) andalso erlang:is_atom(Operation) ->
+    disable_job_class({Application, Operation});
+disable_job_class(Application, Operation) ->
+    {error, {badarg, {Application, Operation}}}.
+
+-spec job_class_enabled(atom(), atom()) -> boolean() | {error, term()}.
+%% @doc Reports whether the specified Application/Operation job class is enabled.
+%% This is the public API for use via RPC.
+job_class_enabled(Application, Operation)
+        when erlang:is_atom(Application) andalso erlang:is_atom(Operation) ->
+    job_class_enabled({Application, Operation});
+job_class_enabled(Application, Operation) ->
+    {error, {badarg, {Application, Operation}}}.
+
+-spec enable_job_class(term()) -> ok | {error, term()}.
+%% @doc Internal API to enable the specified job class.
+%% WARNING:
+%% * This function may not remain in this form once the Jobs API is live!
+%% * Parameter types ARE NOT validated by the same rules as the public API!
+%% You are STRONGLY advised to use enable_job_class/2.
+enable_job_class(Class) ->
```

Comment:
After discussion, we've decided to leave the behavior as-is for 2.2.

Issue RIAK-2744 has been entered to make sure this is addressed if or when we may drive these interfaces programmatically.